kpops/utils/docstring.py

```diff
@@ -21,29 +21,39 @@ def describe_attr(name: str, docstr: str | None) -> str:
     return _trim_description_end(docstr)
 
 
-def describe_class(docstr: str | None) -> str:
-    """Return class description from its docstring
+def describe_object(docstr: str | None) -> str:
+    """Return free-text description from a docstring
 
     Excludes parameters and return definitions
 
     Works only with reStructuredText docstrings.
 
-    :param docstr: The class' docstring
+    :param docstr: The docstring
     :type docstr: str, None
-    :returns: Class description taken from its docstiirng
+    :returns: Description taken from the docstiirng
     :rtype: str
     """
     if docstr is None:
         return ""
+
+    # reST docstrings have a short description/title as their first line.
+    # Optionally, they have a longer description below. Here we separate the
+    # title from the rest as `_trim_description_end()` removes all newlines.
+    docstr = docstr.strip()
+    if "\n" in docstr:
+        title_end = docstr.index("\n")
+        desc = docstr[:title_end] + "\n" + _trim_description_end(docstr[title_end:])
+        return desc.rstrip()
```

Comment:
so this removes the first line of a docstring and then only selects the part below until the attributes section?

Reply:
No, it separates the title from the rest with a newline. Otherwise you end up with descriptions like:

![image](https://user-images.githubusercontent.com/44573859/233384263-20dc0aff-5868-40ce-b9a3-e5c776f394c3.png)

Reply:
makes sense, thanks for explaining